vmas/simulator/environment/environment.py

```diff
@@ -770,6 +774,55 @@ def render(
         # render to display or array
         return self.viewer.render(return_rgb_array=mode == "rgb_array")
 
+    def plot_boundary(self):
+        # Include boundaries in the rendering
+        from vmas.simulator import rendering
+        from vmas.simulator.rendering import Line
+        from vmas.simulator.utils import Color
+
+        # Check boundary limits
+        if self.world.x_semidim is not None or self.world.y_semidim is not None:
+            # Get the origin coordinates
+            origin_x, origin_y = (0, 0)
```

Comment:
We can remove the origin as this will always be the physical origin in 0,0

Reply:
Ok :)